src/functions/template-tags/event.php

```diff
@@ -100,27 +100,29 @@ function tribe_get_event( $event = null, $output = OBJECT, $filter = 'raw', $for
 
 		$post = false;
 
-		if ( ! $force ) {
-			$cache_post    = get_post( $event );
-			if ( empty( $cache_post ) ) {
-				return null;
-			}
+		/** @var Tribe__Cache $cache */
+		$cache = tribe( 'cache' );
+
+		$cache_post    = get_post( $event );
+		if ( empty( $cache_post ) ) {
+			return null;
+		}
+
+		$key_fields    = [
+			$cache_post->ID,
+			$cache_post->post_modified,
+			// Use the `post_password` field as we show/hide some information depending on that.
+			$cache_post->post_password,
+			// We must include options on cache key, because options influence the hydrated data on the Event object.
+			wp_json_encode( Tribe__Settings_Manager::get_options() ),
+			wp_json_encode( [ get_option( 'start_of_week' ), get_option( 'timezone_string' ), get_option( 'gmt_offset' ) ] ),
+			$output,
+			$filter
+		];
 
-			$key_fields    = [
-				$cache_post->ID,
-				$cache_post->post_modified,
-				// Use the `post_password` field as we show/hide some information depending on that.
-				$cache_post->post_password,
-				// We must include options on cache key, because options influence the hydrated data on the Event object.
-				wp_json_encode( Tribe__Settings_Manager::get_options() ),
-				wp_json_encode( [ get_option( 'start_of_week' ), get_option( 'timezone_string' ), get_option( 'gmt_offset' ) ] ),
-				$output,
-				$filter
-			];
-
-			$cache_key     = 'tribe_get_event_' . md5( wp_json_encode( $key_fields ) );
-			/** @var Tribe__Cache $cache */
-			$cache = tribe( 'cache' );
+		$cache_key     = 'tribe_get_event_' . md5( wp_json_encode( $key_fields ) );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 5 spaces

<sub>hash: 48d3bd5442538475a423225b36ace0a0</sub>